core/sections/clause_16_functions.adoc

```diff
@@ -11,69 +11,184 @@
+* s_covers(Geometry a, Geometry b) bool
+* s_coveredBy(Geometry a, Geometry b) bool
+
 ===== Class Geometry
 
 ===== Class Point
 
+* coordinates: double[2..*]
+
 ===== Class MultiPoint
 
+* points: Point[1..*]
+
 ===== Class BoundingBox
 
+* lowerBound: Point
+* upperBound: Point
+
 ===== Class LineString
 
+* points: Point[2..*]
+
 ===== Class MultiLineString
 
+* lineStrings: LineString[1..*]
+
 ===== Class PolygonContour
 
+* points: Point[3..*]
+
 ===== Class Polygon
 
+* outer: PolygonContour
+inner: PolygonContour[0..*]
+
 === Temporal Relation Functions
 ==== Overview
 
 ===== Class Month
 
+* january
+* february
+* march
+* april
+* may
+* june
+* july
+* august
+* september
+* october
+* november
+* december
+
 ===== Class Date
 
+* year: int
+* month: Month
+* day: int
+
 ===== Class TimeOfDay
 
+* hour: int
+* minutes: int
+* seconds: double
+
 ===== Class TimeInstant
 
+* date: Date
+* time: TimeOfDay
+
 ===== Class TimeInterval
 
+* start: TimeInstant
+* end: TimeInstant
+
 ===== Standard TemporalRelationFunctions
 
+* t_after(TimeInstant a, TimeInstant b) bool
+* t_before(TimeInstant a, TimeInstant b) bool
+* t_disjoint(TimeInstant a, TimeInstant b) bool
+* t_equals(TimeInstant a, TimeInstant b) bool
+* t_intersects(TimeInstant a, TimeInstant b) bool
+* t_after(TimeInterval a, TimeInterval b) bool
+* t_before(TimeInterval a, TimeInterval b) bool
+* t_disjoint(TimeInterval a, TimeInterval b) bool
+* t_equals(TimeInterval a, TimeInterval b) bool
+* t_intersects(TimeInterval a, TimeInterval b) bool
+* t_contains(TimeInterval a, TimeInterval b) bool
+* t_during(TimeInterval a, TimeInterval b) bool
+* t_finishedBy(TimeInterval a, TimeInterval b) bool
+* t_finishes(TimeInterval a, TimeInterval b) bool
+* t_meets(TimeInterval a, TimeInterval b) bool
+* t_meetBy(TimeInterval a, TimeInterval b) bool
+* t_overlappedBy(TimeInterval a, TimeInterval b) bool
+* t_overlaps(TimeInterval a, TimeInterval b) bool
+* t_startedBy(TimeInterval a, TimeInterval b) bool
+* t_starts(TimeInterval a, TimeInterval b) bool
+
 === Array Relation Functions
 ==== Overview
 
 ===== Standard ArrayRelationFunctions
 
+* a_containedBy(array a, array b) bool
+* a_contains(array a, array b) bool
+* a_equals(array a, array b) bool
+* a_overlaps(array a, array b) bool
+
 === Text Manipulation Functions
 ==== Overview
 
 ===== Class TextManipulationFunctions
 
+* caseInsensitize(string s) : string
+* accentInsensitize(string s) : string
+* lowerCase(string s) : string
+* upperCase(string s) : string
+* concatenate(string a, string b) : string
+* substitute(string s, string a, string b) : string
+* format(string f, ...) : string
+
 === Geometry Manipulation Functions
 ==== Overview
 
 ===== Standard GeometryManipulationFunctions
 
+* s_intersection(Geometry a, Geometry b) : Geometry
+* s_union(Geometry a, Geometry b) : Geometry
+* s_subtraction(Geometry a, Geometry b) : Geometry
+* s_buffer(Geometry a, double d) : Geometry
+
 ===== Class Geometry
```

Comment:
It looks like we have all the Geometry stuff twice?

We can have a section for it before both the Geometry Manipulation and Geometry relation section. Shouldn't be duplicated.

Reply:
This redundant class has been deleted.